<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PID</class>
 <widget class="QDialog" name="PID">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_read">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Odczytaj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Zapisz</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Zamknij</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_P">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>P:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_I">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>I:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_D">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>D:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_P">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_I">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_D">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_PID">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Regulator PID</string>
   </property>
  </widget>
  <zorder>pushButton_save</zorder>
  <zorder>pushButton_read</zorder>
  <zorder>pushButton_close</zorder>
  <zorder>label_P</zorder>
  <zorder>label_I</zorder>
  <zorder>label_D</zorder>
  <zorder>textEdit_P</zorder>
  <zorder>textEdit_I</zorder>
  <zorder>textEdit_D</zorder>
  <zorder>label_PID</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_close</sender>
   <signal>clicked()</signal>
   <receiver>PID</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
